bash/bashrc.sh

```diff
@@ -0,0 +1,5 @@
+# Note: This file is loaded on all environments, even production.
+
+alias dj="python manage.py"
+alias djrun="python manage.py runserver 0.0.0.0:8000"
+alias djtest="python manage.py test --settings=config.settings.dev"
```

Comment:
If I remember rightly, I tried to introduce something similar last year, but there were just too many cross-platform issues within the team

Reply:
In my case and some other, the `platform sh (psh)` commands  were not getting executed within the container. Perhaps, when @jamesbiggs is back on Monday, he can check too.